docs/index.rst

```diff
@@ -42,6 +42,7 @@ anonymous sources.
    create_admin_account
    test_the_installation
    onboarding
+   offboarding
```

Comment:
Have added to the Topic Guide for now, even though messy, because you're right - it is not part of the installation procedure. Cross-referenced in the Admin Guide in b02ad0159